devtools/create_project/create_project.cpp

```diff
@@ -1499,22 +1499,26 @@ void ProjectProvider::createProject(BuildSetup &setup) {
 	std::string targetFolder;
 
 	if (setup.devTools) {
-		_uuidMap = createToolsUUIDMap();
+		_engineUuidMap = createToolsUUIDMap();
 		targetFolder = "/devtools/";
 	} else if (!setup.tests) {
-		_uuidMap = createUUIDMap(setup);
+		_engineUuidMap = createUUIDMap(setup);
 		targetFolder = "/engines/";
 	}
 
+	_allProjUuidMap = _engineUuidMap;
+
 	// We also need to add the UUID of the main project file.
-	const std::string svmUUID = _uuidMap[setup.projectName] = createUUID(setup.projectName);
+	const std::string svmUUID = _allProjUuidMap[setup.projectName] = createUUID(setup.projectName);
+	// Add the uuid of the detection project
+	const std::string detUUID = _allProjUuidMap["Detection"] = createUUID("Detection");
```

Comment:
Since VS sorts projects alphabetically, it might be more convenient to call this something like `scummvm-detection` (so it's next to it).

Reply:
Good idea! I will incorporate this change.